embassy-traits/src/uart.rs

```diff
@@ -10,6 +10,15 @@ pub enum Error {
 pub trait Uart {
     type ReceiveFuture<'a>: Future<Output = Result<(), Error>>;
     type SendFuture<'a>: Future<Output = Result<(), Error>>;
+    // Receive into the buffer until the buffer is full
```

Comment:
You need to use `///` to create a docstring.
```suggestion
    /// Receive into the buffer until the buffer is full
```